app/components/ui/Navigation.jsx

```diff
@@ -72,8 +72,8 @@ class Navigation extends React.Component {
           </div>
           <ul className={styles.navRight}>
             <li>
-              <a href="https://www.sheltertech.org" target="_blank" rel="noopener noreferrer">
-                About Us
+              <a href="http://help.sfserviceguide.org/" target="_blank" rel="noopener noreferrer">
```

Comment:
Can we use the `https://` link to keep people on the secure, encrypted protocol?